### PR TITLE
Add instructions about pushing changes to setup.py and tagging.

### PR DESCRIPTION
**Pull request by pmdarrow**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -57,6 +57,12 @@ Publishing
 2. Increment version number in ``setup.py`` according to
    `PEP 440 <https://www.python.org/dev/peps/pep-0440/>`_.
 
+3. Commit your change to ``setup.py`` and create a tag for it with the version
+   number. e.g.::
+
+    git tag 0.5.1
+    git push origin 0.5.1
+
 3. Register the package::
```
Comment (DanielArndt) [inline README.rst]: What about step #3?

Comment (pmdarrow) [inline README.rst]: Whooops.

Comment (pmdarrow) [inline README.rst]: Fixed it 👹

**Comment by DanielArndt** [comment]
:+1:

**Pull request merged**